src/index.ts

```diff
@@ -125,6 +125,8 @@ export async function build({
 	let denoVersion =
 		configString(config, 'denoVersion', process.env, 'DENO_VERSION') ||
 		DEFAULT_DENO_VERSION;
+	
+	let denoTsConfig = configString(config, 'denoTsconfig', process.env, 'DENO_TSCONFIG') || undefined;
```

Comment:
I prefer to have it called `tsconfig` rather than `denoTsconfig`:

```suggestion
	const denoTsConfig = configString(config, 'tsconfig', process.env, 'DENO_TSCONFIG');
```